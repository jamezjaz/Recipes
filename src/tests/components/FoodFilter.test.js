import React from 'react';
import renderer from 'react-test-renderer';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import FoodFilter from '../../components/FoodFillter';

afterEach(cleanup);

const foods = [
  {id: 1, name: 'Food name', category: 'Starter'},
  {id: 2, name: 'Food name', category: 'Starter'},
];

it('renders correctly', () => {
  const tree = renderer
    .create(<FoodFilter handleFilter={() => 'mock'} key={foods.id} foods={foods} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});