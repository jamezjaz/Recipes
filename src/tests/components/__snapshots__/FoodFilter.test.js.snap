// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <label
    className="mx-auto my-0"
    htmlFor="select"
  >
    <select
      className="select"
      id="select"
      name="category"
      onChange={[Function]}
    >
      <option
        disable="true"
        hidden={true}
      >
        FOOD CATEGORIES
      </option>
      <option
        value="All"
      >
        All
      </option>
      <option />
    </select>
  </label>
</div>
`;
